### PR TITLE
Add info and blog post about the Phil Bagwell Award for 2019.

### PR DESCRIPTION
**Pull request by sjrd**
/cc @darjutak 

We probably want to add some link/sentence to encourage people to go register for Scala Days, as additional promotion.

@jsuereth We still need a contact address for people to send nominations to. Are you OK with your mail showing up there?

### CODE REVIEW

File: _posts/2019-03-01-preparing-the-bagwell-award-2019.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+  * Activism
+  * Evangelism
+
+Additionally, the committee to select award winners will consist of Martin Odersky and all previous winners of the award (who have time to take part).
+
+The selection process will first involve soliciting nominations from the community.
+Please let us know who you think meets [the published rubric](https://www.scala-lang.org/community/phil-bagwell-award.html) and why.
+The committee will hold voting later in March to select the 2018 and 2019 winners and they will be announced at Scala Days 2019 in Lausanne.
+
+So, please forward all submissions to TODO!
```
Comment (darjutak) [inline _posts/2019-03-01-preparing-the-bagwell-award-2019.md]: add link in place of TODO

File: _posts/2019-03-01-preparing-the-bagwell-award-2019.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
+---
+category: blog
+title: "Preparing the 2019 Phil Bagwell Community Award"
+by: Josh Suereth
+---
+
+Hello Scala Community!
+
+It is almost Scala Days and that means it is time to select Phil Bagwell award winners!
```
Comment (darjutak) [inline _posts/2019-03-01-preparing-the-bagwell-award-2019.md]: link Scala Days to www.scaladays.org

File: _posts/2019-03-01-preparing-the-bagwell-award-2019.md (hunk 1 above)
Comment (sjrd) [reply _posts/2019-03-01-preparing-the-bagwell-award-2019.md]: See my question to Josh in the PR description. ;)

Comment (darjutak) [reply _posts/2019-03-01-preparing-the-bagwell-award-2019.md]: ah, right, we need an actual address, sorry about that

Comment (jsuereth) [reply _posts/2019-03-01-preparing-the-bagwell-award-2019.md]: SO I had that in my original CL: https://github.com/jsuereth/scala-lang/commit/15d91131e3cee357061f1033f2eaf4654d0a5458

Comment (jsuereth) [reply _posts/2019-03-01-preparing-the-bagwell-award-2019.md]: bagwellaward@gmail.com

Comment (jsuereth) [reply _posts/2019-03-01-preparing-the-bagwell-award-2019.md]: Darja made a google form though, that's probably a better way to take in submissions.

**Comment by SethTisue** [comment]
+1 to using a form

**Comment by sjrd** [comment]
Updated with a link to the form.

File: _posts/2019-03-01-preparing-the-bagwell-award-2019.md [hunk 3]
```diff
@@ -0,0 +1,31 @@
+---
+category: blog
+title: "Preparing the 2019 Phil Bagwell Community Award"
```
Comment (darjutak) [inline _posts/2019-03-01-preparing-the-bagwell-award-2019.md]: please change the title to: "Phil Bagwell Nominations"

Comment (sjrd) [reply _posts/2019-03-01-preparing-the-bagwell-award-2019.md]: Done.

**Pull request merged**